src/mscorlib/src/System/Runtime/Intrinsics/X86/Bmi2.cs

```diff
@@ -49,6 +49,6 @@ public static class Bmi2
         /// <summary>
         /// unsigned __int64 _pext_u64 (unsigned __int64 a, unsigned __int64 mask)
         /// </summary>
```

Comment:
nit: missing instruction

Reply:
I didn't update any of the BMI, AES, LZCNT, or POPCNT ISAs (looks like this picked up a trailing whitespace removal).

I'll update these real quick.

Reply:
Fixed.